config/routes.rb

```diff
@@ -7,4 +7,10 @@
 
   mount JasmineRails::Engine => '/specs' if defined?(JasmineRails)
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'contact-me', to: 'messages#new', as: 'new_message'
+  # ^ creates a route named new_message_url which maps GET requests
```

Comment:
use resources. also this is pretty but make the route /contact, it'll be easier and look cleaner